recipes/org-analyzer

```diff
@@ -0,0 +1,3 @@
+(org-analyzer :fetcher github
+              :repo "rksm/clj-org-analyzer"
+              :files ("org-analyzer-el/*.el"))
```

Comment:
This recipe looks like it will include `org-analyzer-pkg.el`, which it shouldn't.